demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1358,8 +1365,18 @@ export class DemoMeetingApp
       await Promise.all([
         this.createLogStream(configuration, 'create_log_stream'),
         this.createLogStream(configuration, 'create_browser_event_log_stream'),
+        this.createLogStream(configuration, 'create_browser_event_ingestion_log_stream'),
       ]);
 
+      const eventReportingPOSTLogger = new MeetingSessionPOSTLogger(
```

Comment:
nit: Can we move it down below the other 2 so the order matches with the above init?

Reply:
Update, moved below the existing `meetingSessionPOSTLogger`. Thanks!